libs/platform/ebpf_program_types.c

```diff
@@ -43,7 +43,7 @@ ebpf_program_information_decode(
 
     memcpy(local_buffer, buffer, buffer_size);
 
-    RPC_STATUS status = MesDecodeBufferHandleCreate((char*)local_buffer, buffer_size, &handle);
+    RPC_STATUS status = MesDecodeBufferHandleCreate((char*)local_buffer, (unsigned long)buffer_size, &handle);
```

Comment:
Do we want to check for truncation and return an error here? We are truncating from 64bit -> 32bit.

Reply:
done